src/app/shipyard/Module.js

```diff
@@ -44,13 +44,7 @@ export default class Module {
   getModValue(name, raw) {
     let result = this.mods  && this.mods[name] ? this.mods[name] : null;
 
-    // Calculate the percentage change for a synthetic value
-    if (STATS_FORMATING[name] && STATS_FORMATING[name].synthetic) {
-      const statGetter = this[STATS_FORMATING[name].synthetic];
-      let unmodifiedStat = statGetter.call(this, false);
-      let modifiedStat = statGetter.call(this, true);
-      result = (modifiedStat / unmodifiedStat - 1)  * 10000;
-    } else if ((!raw) && this.blueprint && this.blueprint.special) {
+    if ((!raw) && this.blueprint && this.blueprint.special) {
```

Comment:
why the double parenthesis around raw?

Reply:
Actually I never touched that code.

Reply:
I wouldn't change this one as it's a wasted commit.